lib/rubocop/packs.rb

```diff
@@ -13,66 +15,38 @@ module Packs
     PACK_LEVEL_RUBOCOP_YML = 'package_rubocop.yml'
     PACK_LEVEL_RUBOCOP_TODO_YML = 'package_rubocop_todo.yml'
 
-    class Error < StandardError; end
-    extend T::Sig
-
-    # Your code goes here...
     PROJECT_ROOT   = T.let(Pathname.new(__dir__).parent.parent.expand_path.freeze, Pathname)
     CONFIG_DEFAULT = T.let(PROJECT_ROOT.join('config', 'default.yml').freeze, Pathname)
     CONFIG         = T.let(YAML.safe_load(CONFIG_DEFAULT.read).freeze, T.untyped)
 
     private_constant(:CONFIG_DEFAULT, :PROJECT_ROOT)
 
-    class Offense < T::Struct
-      extend T::Sig
-
-      const :cop_name, String
-      const :filepath, String
-
-      sig { returns(ParsePackwerk::Package) }
-      def pack
-        ParsePackwerk.package_from_path(filepath)
-      end
-    end
-
-    sig { params(paths: T::Array[String], cop_names: T::Array[String]).returns(T::Array[Offense]) }
-    def self.offenses_for(paths:, cop_names:)
```

Comment:
I moved this to `Private` because I didn't end up needing it in the public API. I originally intended for `use_packwerk` to use it to see if it wanted to add any files to the TODO, but realized it is a bit awkward to have use packwerk get offenses only to pass them back into some new function to add them to the TODO. I realized it was simpler to just tell rubocop packs to add any new TODOs directly.